### ISSUE TITLE
How to add Animate.css to carousel content 

### ISSUE THREAD

**Issue by cptacemaverick** [smashingboxes/OwlCarousel2#701]
How do i add animation to carousel content that triggers on the carousel changing or being clicked to the next item?

**Comment by pajasevi** [comment]
```javascript
$owl.on('change.owl.carousel', function( event ) {
  // trigger your animation function
});
```
Do you mean something like this? You can get index of item from the event: event.item.index

**Comment by cptacemaverick** [comment]
So say i want to add an animated property class to my <H1> on change?

**Comment by pajasevi** [comment]
Yes, that's it. (well, you are adding class to #header, not h1, but it would work) And of course you have to store your carousel in $owl variable beforehand.

**Comment by cptacemaverick** [comment]
How do I do that?

**Comment by pajasevi** [comment]
You have to store it in variable and then initialize OwlCarousel using the variable like this:
```javascript
var $owl = $('.your-carousel');
$owl.owlCarousel();
```
Then you can use event listening, like in my example.

**Comment by cptacemaverick** [comment]
Ok thanks man ill give that a try

**Comment by t3premium** [comment]
This isn't working infinite. Look at this example of owl.carousel 1: http://wpcheatsheet.net/infinite-animated-css-in-owl-carousel/ You had to remove class animated after change but this function is missing in owl.carousel 2: afterMove and beforeMove

**Comment by mikesheridan** [comment]
I was having a similar issue in trying to add a css animation to a div within my owl slide.

I discovered by watching chrome dev tools that owl adds an active class in the dom as the slides rotate.

I was able to trigger an animation by specifying the active parent node and my target div to attach the animation:

div.active div.hero_box {  
  animation-name: moveInUp;
  animation-duration: 0.6s;
 animation-delay: 0.3s;
 animation-timing-function: ease;
 animation-fill-mode: both;
 backface-visibility: hidden;
}

@keyframes moveInUp {
  0% {
    opacity: 0;
    transform: translateY(50px);
  }
  100% {
    opacity: 1;
    transform: translateY(0);
  }
}

This worked for me and my animation is firing every time the slide changes.

Hope that helps.

Mike
